presto-hive/src/main/java/io/prestosql/plugin/hive/s3/HiveS3Module.java

```diff
@@ -45,6 +45,9 @@ else if (type == S3FileSystemType.EMRFS) {
             validateEmrFsClass();
             newSetBinder(binder, ConfigurationInitializer.class).addBinding().to(EmrFsS3ConfigurationInitializer.class).in(Scopes.SINGLETON);
         }
+        else if (type == S3FileSystemType.CUSTOM) {
+            binder.bind(ConfigurationInitializer.class).to(CustomFsS3ConfigurationUpdater.class).in(Scopes.SINGLETON);
```

Comment:
Is that required? Will Guice fail when you don't bind anything? `io.prestosql.plugin.hive.HdfsConfigurationInitializer#HdfsConfigurationInitializer(io.prestosql.plugin.hive.HiveConfig, java.util.Set<io.prestosql.plugin.hive.s3.ConfigurationInitializer>)` accepts `Set`, so possibly you don't need to bind anything